src/stores/PreferenceStore.ts

```diff
@@ -107,12 +107,12 @@ const DEFAULTS = {
         contourColormap: "viridis",
     },
     WCS_OVERLAY: {
-        astColor: 4,
+        astColor: "auto-blue",
```

Comment:
I think we'll need to create a new preference schema (version 2) and will need to also add an upgrade path for going from v1 (astColor: number) -> v2 (astColor: string)

Reply:
It _might_ be easier to rename this setting to overlayColor in the new schema, as this would prevent conflicts